$NetBSD$

fts_set is not detected because NetBSD has renamed the symbol.
Remove the check and assume it is available.

--- CMakeLists.txt.orig	2020-06-16 17:19:13.000000000 +0000
+++ CMakeLists.txt
@@ -247,19 +247,19 @@ else()
 		set( ALL_C_FLAGS "-ffp-contract=off -DHAVE_SOFTPOLY" )
 	endif()
 
-	if ( UNIX )
-		include(CheckSymbolExists)
-		check_symbol_exists( "fts_set" "fts.h" HAVE_FTS )
-		if ( NOT HAVE_FTS )
-			include ( FindPkgConfig )
-			pkg_check_modules( MUSL_FTS musl-fts )
-			if ( MUSL_FTS_FOUND )
-				set ( ALL_C_FLAGS "${ALL_C_FLAGS} ${MUSL_FTS_LDFLAGS}" )
-			else ( MUSL_FTS_FOUND )
-				message (ERROR "fts_* functions not found in the system" )
-			endif ( MUSL_FTS_FOUND )
-		endif ( NOT HAVE_FTS )
-	endif ( UNIX )
+#	if ( UNIX )
+#		include(CheckSymbolExists)
+#		check_symbol_exists( "fts_set" "fts.h" HAVE_FTS )
+#		if ( NOT HAVE_FTS )
+#			include ( FindPkgConfig )
+#			pkg_check_modules( MUSL_FTS musl-fts )
+#			if ( MUSL_FTS_FOUND )
+#				set ( ALL_C_FLAGS "${ALL_C_FLAGS} ${MUSL_FTS_LDFLAGS}" )
+#			else ( MUSL_FTS_FOUND )
+#				message (ERROR "fts_* functions not found in the system" )
+#			endif ( MUSL_FTS_FOUND )
+#		endif ( NOT HAVE_FTS )
+#	endif ( UNIX )
 
 	set( REL_C_FLAGS "" )
 	set( DEB_C_FLAGS "" )
